kt-extensions/src/main/kotlin/com/mindera/skeletoid/kt/extensions/rxjava/Maybe.kt

```diff
@@ -0,0 +1,18 @@
+package com.mindera.skeletoid.kt.extensions.rxjava
+
+import io.reactivex.Maybe
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+
+inline fun <T : Any> Maybe<T>.subscribeIO(): Maybe<T>
```

Comment:
does it need to be `inline` ?

Reply:
I guess it doesn't need to be `inline`. We just added it to make it consistent with the other extensions we already have for other types of observables. We should probably remove the `inline` from them all. What's your opinion about it?

Reply:
Since it doesn't have anything like `reified` which needs context of where it is called from and the params aren't lambdas either, `inline` can actually hurt on the long run and have the reverse effect and hurt performance. It is a small operation so it would be worse if it was a bigger block, so I'm not totally against it, but still, I follow the rule of thumb that if it isn't either of those cases, I drop the `inline`. Also I think studio gives a warning saying `inline` shouldn't be used in this situation since the trade-off isn't worth it.

Reply:
Just seen your reply @djrsousa - yes, if the others are like that I guess they should be removed as well 👍